content/tokio/topics/tracing.md

```diff
@@ -75,6 +75,54 @@ pub async fn main() -> mini_redis::Result<()> {
 If you run your application now, you may see some trace events emitted by tokio,
 but you will need to modify your own application to emit traces to get the most
 out of `tracing`.
+##  Subscriber Configuration
```

Comment:
nit: i think there should be an extra line of whitespace here --- i think that was my fault
```suggestion
out of `tracing`.

##  Subscriber Configuration
```